src/components/10-atoms/link/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@axa-ch/link",
+  "version": "1.0.6",
+  "description": "The axa link for internal and external page navigation.",
+  "author": "Pattern Warriors",
+  "homepage": "https://github.com/axa-ch/patterns-library/blob/develop-v2/src/components/10-atoms/link/README.md",
+  "license": "Copyright 2019 AXA Versicherungen AG",
+  "main": "lib/index.js",
+  "directories": {
+    "lib": "lib"
```

Comment:
do we need directories?

Reply:
I have no idea tbh. The doc states this:

directories.lib§
Tell people where the bulk of your library is. Nothing special is done with the lib folder in any way, but it’s useful meta info.

Reply:
Sounds optional, I just made a tasty copypasta and didn't think further about it.

Reply:
ok